lib/auth/authclient/authclient.go

```diff
@@ -0,0 +1,113 @@
+/*
+Copyright 2022 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package authclient contains common code for creating an auth server client
+// which may use SSH tunnelling through a proxy.
```

Comment:
Turns out this is another difference US vs Canadian/UK English, huh. I will change it since code usually goes with the US spelling and I'm not feeling very colonial today